packages/common/src/core/schemas/shared/CatalogSchema.ts

```diff
@@ -82,7 +82,37 @@ export const GalleryDisplayConfigSchema: IConfigurationSchema = {
   type: "object",
   properties: {
     hidden: { type: "boolean", default: false },
-    // TODO: fill in properties
+    layout: {
+      type: "string",
+      enum: ["list", "grid", "table", "map", "compact"],
+      default: "list",
+    },
+    cardTitleTag: {
+      type: "string",
+      enum: ["h1", "h2", "h3", "h4", "h5", "h6"],
+      default: "h3",
+    },
+    // showThumbnail can be either a boolean or a string,
+    // if it's a string, it can be "grid" which means show
+    // the thumbnail in grid layout only
+    showThumbnail: {
+      oneOf: [{ type: "string" }, { type: "boolean" }],
+      enum: [true, false, "grid"],
+      default: true,
+    },
+    corners: { type: "string", enum: ["square", "round"], default: "square" },
+    shadow: {
+      type: "string",
+      enum: ["none", "low", "medium", "heavy"],
+      default: "none",
+    },
```

Comment:
That's a good point. I updated to add/use some enums but not all of them. Because these will later be used for props passed into the gallery and the types for the same props that the gallery uses are defined in `hub-components`, e.g. [here](https://github.com/ArcGIS/opendata-ui/blob/master/packages/hub-components/src/components/interfaces.ts) and [here](https://github.com/ArcGIS/opendata-ui/blob/master/packages/hub-components/src/components/arcgis-hub-gallery/arcgis-hub-gallery.tsx#L27-L28), using different types here will cause errors, and updating types in the gallery might break the existing use cases. I hope this makes sense.